saveutils/tools/sizeanalysis.py

```diff
@@ -10,8 +11,9 @@
 logger = getLogger(__name__)
 logger.setLevel("INFO")
 handler = StreamHandler()
-handler.setFormatter(Formatter("[%(asctime)s][%(name)s][%(levelname)s] %(message)s"))
+handler.setFormatter(Formatter("[%(asctime)s][%(name)s] %(message)s", "%m-%d %H:%M:%S"))
```

Comment:
This was to make the log line shorter. As it stands the leader on each line is a significant portion of each line. That said, I can combine all the lines into one string and print them once.

Reply:
For the record, I'm not heavily attached to this change, so it can be dropped entirely if it doesn't fit your desired style. With that said, if you want a consistent style across the program, you should probably have a single Formatter that everything references.

Reply:
I'd prefer we drop this for now, in order to keep logging style uniform across all tools, but a global formatter / general logging overhaul is a good idea for a future update.

Reply:
reverted in 7626b47
formatting cleaned up in the following commits, it now results in output like this:

```
P:\-2-\SOD-SaveUtils\
> python .\saveutils\cli.py -i $save sizeanalysis
[2023-06-01 20:01:31,270][tools.sizeanalysis][INFO] Analysing save file size...
[2023-06-01 20:01:31,839][tools.sizeanalysis][INFO] Total save file size (APPROX): 11,100,000 bytes
[2023-06-01 20:01:31,839][tools.sizeanalysis][INFO] Estimated fudge factor is 114.4436%
[2023-06-01 20:01:32,124][tools.sizeanalysis][INFO]
interactables                      5,370,000 bytes /  48.378%
citizens                           2,810,000 bytes /  25.315%
evidence                           1,470,000 bytes /  13.243%
messageThreads                     1,060,000 bytes /   9.550%
rooms                                233,000 bytes /   2.099%
basicJobs                             95,900 bytes /   0.864%
doors                                 40,500 bytes /   0.365%
companies                              8,190 bytes /   0.074%
buildings                              6,490 bytes /   0.058%
addresses                              5,810 bytes /   0.052%
footprints                             2,810 bytes /   0.025%
stolenItemJobs                         2,740 bytes /   0.025%
sum of items < 0.005%                  5,164 bytes /   0.047%
NOTE: due to conversions between raw strings and python, sizes are estimates only.
```